src/multidim/intervalbox.jl

```diff
@@ -109,3 +109,28 @@ IntervalBox(x::Interval, n::Int) = IntervalBox(x, Val(n))
 dot(x::IntervalBox, y::IntervalBox) = dot(x.v, y.v)
 
 Base.:(==)(x::IntervalBox, y::IntervalBox) = x.v == y.v
+
+
+"""
+    mince(x::IntervalBox, n)
+
+Splits `x` in `n` intervals in each dimension of the same diameter. These
```

Comment:
For higher dimensions this is a bad idea. Is this version actually useful?

Reply:
I guess you mean that things grow as N^n; if so, I agree that this may not be ideal. 

I think this may be indeed useful: my motivation comes from the computation of tighter bounds for `TaylorModelN`'s when computing a validated integration. In some cases, e.g., when you have to check that certain region is not crossed this is possibly cheaper than reducing other parameters (e.g. integration step), in the sense of the *very many* operations (and memory) involved in taking the different stepsizes